src/Storage/Storage.Management/help/Get-AzRmStorageShare.md

```diff
@@ -157,6 +157,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -Filter
```

Comment:
Please add an example for filter usage.
Since the usage is not so simple, we should have an example to show how to use it.

Reply:
Added